resources/js/views/Diagnostics.vue

```diff
@@ -28,9 +28,35 @@ import SpaceDiagnostics from "@/components/diagnostics/SpaceDiagnostics.vue";
 import { useAuthStore } from "@/stores/Auth";
 import { storeToRefs } from "pinia";
 import { useTogglablesStateStore } from "@/stores/ModalsState";
+import { useToast } from "primevue/usetoast";
 
 const authStore = useAuthStore();
 const { user } = storeToRefs(authStore);
 authStore.getUser();
 const togglableStore = useTogglablesStateStore();
+
+const toast = useToast();
+const errorLoaded = ref<boolean>(false);
+const infoLoaded = ref<boolean>(false);
+const configurationLoaded = ref<boolean>(false);
+
+const canCopy = computed(() => errorLoaded.value && infoLoaded.value && configurationLoaded.value);
+
+function copy() {
+	const errors = document.getElementById("ErrorsData");
```

Comment:
Whoa, that's definitely not the way to do it! 💩 

it is possible to transmit the data to the outside via the emit
https://vuejs.org/guide/typescript/composition-api#typing-component-emits

Reply:
That's still not the way to do it. The other is more standardized 😄